todo_rxbloc_app/lib/feature_homepage/bloc/navigation_bloc.dart

```diff
@@ -25,9 +25,9 @@ class NavigationBloc extends $NavigationBloc {
   Stream<NavigationEnum> _mapToGetPageIndexState() =>
       _$setPageIndexEvent.doOnData((event) {
         if (event == NavigationEnum.todosList) {
-          router.go(const TodoListRoute().location);
```

Comment:
Hmm, why we don't use routes directly anymore ?

Reply:
If by using routes directly you mean  like `TodoListRoute().go(context)`  <- this requires context, which I do not have inside the bloc